README.md

```diff
@@ -1,9 +1,16 @@
 # Giggle Click
 
 ## Designed and Developed by:
+
 - [Peter](https://github.com/PJSalter)
 - [Milly](https://github.com/millipede-cpu)
 
+## Getting started
```

Comment:
Thank you for adding this, it is a great reminder on what commands to use, very helpful to look back at ✨